src/demo/README.md

```diff
@@ -0,0 +1,58 @@
+## Prepare
+### Run docker-compose
+   It'll start:
+   - Postgres db for the VCAuthn service
+   - Keycloak with a pre-configured vc-authn realm
+
+#### Start VCAuthn service
```

Comment:
Is this meant to be under Run docker-compose?
### vs ####

Reply:
That wasn't my intent. Service needs to be started separately from the docker compose. The idea was to list a sequence of steps necessary to set up a test env.